schema-generic.graphql

```diff
@@ -292,24 +231,9 @@ type Pool @entity @regularPolling {
   " Symbol of liquidity pool (e.g. 3CRV) "
   symbol: String
 
-  # Generally protocols accept one or multiple tokens and mint tokens to the depositor to track ownership
-  # Some protocols don't mint any tokens to track ownership, in that case outputToken is null.
-
-  " Token that is minted to track ownership of position in protocol "
-  outputToken: Token
```

Comment:
I am thinking we should keep outputToken. I think it's a field that a lot of pools have and it's generally good info

Reply:
Added back!